frontend/public/style/_status.scss

```diff
@@ -7,21 +7,21 @@
 }
 
 .co-m-status--critical {
-  color: $red-dark;
+  color: $color-pf-red-200 ;
```

Comment:
`$red-invalid` and `$red-dark` are both(?) being changed to `$color-pf-red-200`
If `$red` becomes `$color-pf-red` which is `$color-pf-red-300`, then shouldn't `$red-dark` become `$color-pf-red-400`